### PR TITLE
User disabled date

### PR DESCRIPTION
**Pull request by brig**

### CODE REVIEW

File: server/dist/src/main/resources/concord-server.conf [hunk 1]
```diff
@@ -450,6 +450,9 @@ concord-server {
 
         # interval for the automatic group sync (interval)
         minAgeSync = "1 day"
+
+        # interval to delete disabled users (interval)
```
Comment (amithkb) [inline server/dist/src/main/resources/concord-server.conf:454]: If we want to disable `delete` old user.. then `disabledAge` should be set to some large?

Comment (ibodrov) [reply server/dist/src/main/resources/concord-server.conf:454]: That's actually a good question. I also don't think that this feature should be enabled by default.

Comment (brig) [reply server/dist/src/main/resources/concord-server.conf:454]: > If we want to disable delete old user.. then disabledAge should be set to some large one?

just set `disabledAge` to null...

Comment (brig) [reply server/dist/src/main/resources/concord-server.conf:454]: > That's actually a good question. I also don't think that this feature should be enabled by default.
fixed. now this functionality disabled by default...

**Review by ibodrov — APPROVED** [review]

**Pull request merged**